tools/ratbagd.py

```diff
@@ -303,6 +305,20 @@ def capabilities(self):
         return self._get_dbus_property("Capabilities")
 
     @GObject.property
+    def name(self):
+        """The name of the profile"""
+        return self._get_dbus_property("Name")
+
+    @name.setter
+    def name(self, name):
+        """Set the name of this profile.
+
+        @param name The new name, as string"""
+        if RatbagdProfile.CAP_WRITABLE_NAME in self.capabilities:
+            return self._set_dbus_property("Name", "s", name)
+        return RatbagErrorCode.RATBAG_ERROR_CAPABILITY
```

Comment:
OK, I finally understood why raising an exception in the setter doesn't work. We are using GObjects, and according to https://stackoverflow.com/questions/19639089/no-exceptions-from-gobject-properties-in-pygobject GObject doesn't support exceptions, and so there is a try/catch in the setter.

After looking at the code of GObject.Property, it seems we can force the exception to be raised as in the updated commit below.